lib/index.js

```diff
@@ -14,6 +14,7 @@ const formatOptions = require('./options-formatter');
 const generateSVG = require('./generate-svg');
 const generateStyles = require('./generate-styles');
 const { stripVariables, findVariables } = require('./variable-parser');
+const generateSpritePrefix = require('./helpers/generate-sprite-prefix');
```

Comment:
The helper is not being used in this file yet?

Reply:
And they seem to still be missing from the [style formatters](https://github.com/cascornelissen/svg-spritemap-webpack-plugin/tree/master/lib/style-formatters) (like [overe here](https://github.com/cascornelissen/svg-spritemap-webpack-plugin/blob/master/lib/style-formatters/scss.js#L44) for SCSS)?